Samples/Windowing/cs-winforms-unpackaged/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Windows.Forms;
+using Microsoft.Windows.ApplicationModel;
+
+namespace winforms_unpackaged_app
+{
+    static class Program
+    {
+        /// <summary>
+        ///  The main entry point for the application.
+        /// </summary>
+        [STAThread]
+        static void Main()
+        {
+            Application.EnableVisualStyles();
+            Application.SetCompatibleTextRenderingDefault(false);
+            
+            // Initialize dynamic dependencies so we can consume the Windows App SDK APIs from this unpackaged app.
+            // Take a dependency on Windows App SDK v1.0.0-preview1.
+            int hr = MddBootstrap.Initialize(0x00010000, "preview2");
```

Comment:
Should this be updated to the stable release? 
Also, since preview3 there's a way to avoid using the MddBootstrapper (although what you currently have here is still valid if more control is needed over initialization)
See this, for example:
https://github.com/microsoft/WindowsAppSDK-Samples/blob/94a9c87565958c68fa4f2737629199ec73673e22/Samples/Unpackaged/README.md#using-the-windows-app-sdk-in-an-unpackaged-desktop-app

Reply:
There was winforms designer problem with preview3, which is why the sample is still in preview2. It should not be an issue for GA. We can wait to add the sample until GA.